### DISCUSSION TITLE
Time for newer Winget Preview?

### DISCUSSION THREAD

**Discussion by KnifMelti** [Romanitho/Winget-Install-GUI#46] [Ideas]
Why `if (!($AppInstallerVers -gt "1.18.0.0"))` when the current preview now is so much better (install from Store without account, etc)?
Why not `if (!($AppInstallerVers -gt "1.18.1251.0"))` or `if (!($AppInstallerVers -eq "1.18.1391.0"))`?

**Comment by Romanitho** [comment]
it is already the lastest in WiGui, but you're right, i'll match the check :) thanks

**Reply by KnifMelti** [reply]
But, with your current just released check -gt 1391 it will keep installing all the time, or...?

**Reply by Romanitho** [reply]
changed on latest version. I realized that as well :)
